src/strands/agent/agent.py

```diff
@@ -285,15 +284,18 @@ def __init__(
         self.record_direct_tool_call = record_direct_tool_call
         self.load_tools_from_directory = load_tools_from_directory
 
-        self.tool_registry = ToolRegistry()
+        # Initialize the tool registry with the directory loading option
+        self.tool_registry = ToolRegistry(load_tools_from_directory=self.load_tools_from_directory)
```

Comment:
Since we're changing this interface can we also allow the directories to be passed for watching ^^

Idea is we're only allowing `./tools` directory to be loaded but customers can pass their directories like:

`Agent(tool_directories=["./tools", -> default, ...]`